### ISSUE TITLE
wrap pid controller for python with swig

### ISSUE THREAD

**Issue by knmcguire** [bitcraze/crazyflie-simulation#3]

**Comment by knmcguire** [comment]
Implemented in 8aff66040672e1cf8b218c23b56578f26a12a51d

Currently for webots i get the following error for the python controller:

`WARNING: The current physics step could not be computed correctly. Your world may be too complex. If this problem persists, try simplifying your bounding object(s), reducing the number of joints, or reducing WorldInfo.basicTimeStep`

Weird thing is that I am not seeing this at all for the c controller, and I really replicated the code step by step. 

I already asked a question to the webots tech service about this on discord.

**Comment by knmcguire** [comment]
yeah.... it was because the rotors were not going in the right direction... classic. I first did it without the minuses like this.

```
    m1_motor.setVelocity(-motorPower.m1)
    m2_motor.setVelocity(motorPower.m2)
    m3_motor.setVelocity(-motorPower.m3)
    m4_motor.setVelocity(motorPower.m4)
```
probably a bit too much context switching between webotts and gazebo!
